omod/src/main/webapp/fragments/field/maritalStatus.gsp

```diff
@@ -6,8 +6,8 @@
     <select name="${config.formFieldName}">
```

Comment:
@slubwama This is the style that I am advocating for to ease the burden on the data entry staff who follow the codes